sfaira/data/dataloaders/loaders/d10_1038_s41591_018_0096_5/human_lung_2018_NA_Lambrechts_001.yaml

```diff
@@ -0,0 +1,43 @@
+dataset_structure:
+    dataset_index: 1
+    sample_fns:
+dataset_wise:
+    author: "Lambrechts, Diether"
+    doi: "10.1038/s41591.018.0096.5"
+    download_url_data: "https://bit.ly/387ZM2s"
+    download_url_meta: "https://bit.ly/3dGyjGr"
+    normalization: "raw"
+    year: "2018"
+dataset_or_observation_wise:
+    age:
+    age_obs_key:
+    assay: "NA"
+    assay_obs_key:
+    bio_sample:
+    bio_sample_obs_key: 
+    development_stage:
+    development_stage_obs_key:
+    ethnicity:
+    ethnicity_obs_key:
+    healthy: "False"
+    healthy_obs_key:
+    individual:
+    individual_obs_key: "PatientNumber"
+    organ: "lung"
+    organ_obs_key:
+    organism: "human"
+    organism_obs_key:
+    sex:
+    sex_obs_key:
+    state_exact: "cancer"
+    state_exact_obs_key:
+    tech_sample:
+    tech_sample_obs_key: "sample"
+observation_wise:
+    cellontology_original_obs_key: "CellType"
+feature_wise:
```

Comment:
you need to supply var_ensembl_col and/or var_symbol_col so that it s clear what the feature dimensions are

Reply:
okay I now have the gene symbols as index of adata.var, should I change that and set it to a column of adata.var?

Reply:
just put `var_symbol_col: "index"`